pkg/types/host/types.go

```diff
@@ -0,0 +1,31 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * 蓝鲸智云 - 配置平台 (BlueKing - Configuration System) available.
+ * Copyright (C) 2017 THL A29 Limited,
+ * a Tencent company. All rights reserved.
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://opensource.org/licenses/MIT
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ * We undertake not to change the open source license (MIT license) applicable
+ * to the current version of the project delivered to anyone in the future.
+ */
+
+// Package host define host types
+package host
+
+import "configcenter/src/common/metadata"
+
+// HostBaseInfo host base info
+type HostBaseInfo struct {
+	BKHostIDField        int64                        `json:"bk_host_id" bson:"bk_host_id"`
+	BKHostNameField      string                       `json:"bk_host_name" bson:"bk_host_name"`
+	BKHostInnerIPField   metadata.StringArrayToString `json:"bk_host_innerip" bson:"bk_host_innerip"`
+	BKHostInnerIPv6Field metadata.StringArrayToString ` json:"bk_host_innerip_v6" bson:"bk_host_innerip_v6"`
```

Comment:
把字段名里的BK都去掉，然后把这3个字段多的空格删掉
```suggestion
	HostInnerIPv6Field metadata.StringArrayToString `json:"bk_host_innerip_v6" bson:"bk_host_innerip_v6"`
```